src/components/Breadcrumb/Breadcrumb.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-unused-vars */
+import { Link, useLocation } from 'react-router-dom';
+import { v4 as uuidv4 } from 'uuid';
+import translit from '../../utils/Translit';
+
+export default function Breadcrumb() {
+  const location = useLocation();
+  const crumbPath = location.pathname.split('/').filter((path) => path !== '');
+
+  const breadCrumb = crumbPath.map((crumb, index) => {
+    const uniqueId = uuidv4();
+    let crumbLink = '';
+    crumbLink += `/${crumb}`;
+
+    return (
+      <li key={uniqueId}>
```

Comment:
```suggestion
      <li key={crumbLink}>
```
key здесь нужен, чтобы реакт понимал какие элементы списка перерендеривать. Теперь на каждый рендер сюда будут попадать новые значения, и реакт поэтому будет думать, что надо все перерендеривать
https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key